instrumentation/grape/opentelemetry-instrumentation-grape.gemspec

```
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/instrumentation/grape/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-instrumentation-grape'
+  spec.version     = OpenTelemetry::Instrumentation::Grape::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'Grape instrumentation for the OpenTelemetry framework'
+  spec.description = 'Grape instrumentation for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = Dir.glob('lib/**/*.rb') +
+               Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.7.6'
+
+  spec.add_dependency 'activesupport'
```

We must not introduce transitive runtime dependencies to 3rd party libraries. If the `ActiveSupport` is not defined, then they should not be able to use this instrumentation. 

```suggestion
  spec.add_development_dependency 'activesupport'
```

---

Done ✅